integration/testnet/network.go

```diff
@@ -576,11 +576,16 @@ func PrepareFlowNetwork(t *testing.T, networkConf NetworkConfig) *FlowNetwork {
 
 		// if node is of LN/SN role type add additional flags to node container for secure GRPC connection
 		// this is required otherwise collection node will fail to startup
+		nodeContainer := flowNetwork.Containers[nodeConf.ContainerName]
+
+		// testnet is running at a significantly smaller scale than a production network, hence to
+		// hold the mathematics behind topology connectivity valid, we need a faster connection update rate than the default value:
+		nodeContainer.AddFlag("peerupdate-interval", "1s")
```

Comment:
@yhassanzadeh13 Could you add the flag here instead: https://github.com/onflow/flow-go/blob/8f5f429ff969444c527e6a73b1474a37f6c7f18f/integration/testnet/network.go#L671.

In `AddNode` is the standard place where we add all config flags. The only reason `access-node-ids` is added here instead, is because we need two know all AN addresses to set the flag, and this isn't guaranteed to be known in `AddNode`.

Reply:
Addressed in https://github.com/onflow/flow-go/pull/1949/commits/994d7b001a57b6e20598431e2c30c66b1f7db1bc